pipelines/pepatac.yaml

```diff
@@ -27,6 +27,7 @@ tools:  # absolute paths to required tools
 # user configure 
 resources:
   genomes: ${GENOMES}
+  genome_config: ${REFGENIE}
```

Comment:
ultimate this will eliminate the need for the `genomes` key, right?

Reply:
Yes